### PR TITLE
Add blog post for archive 1.1.0

### PR DESCRIPTION
**Pull request by jimhester**

### CODE REVIEW

**Comment by jimhester** [comment]
Does anyone think we need any tweaks to this, or should I merge it as-is?

File: content/blog/archive-1-1-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+output: hugodown::hugo_document
+
+slug: archive-1-1-0
+title: archive 1 1 0
+date: 2021-08-23
```
Comment (jennybc) [inline content/blog/archive-1-1-0/index.Rmd:6]: These need updating.

File: content/blog/archive-1-1-0/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+archive_write_files("nycflights13.zip", fs::dir_ls("nycflights13"))
+```
+
+```{r}
+archive("nycflights13.zip")
+```
+
+## Reading single files from an archive
+
+`archive_read()` is used to read a single file from an archive. This function returns an R connection, which can be passed to many R functions that takes a connection object as input. All base R file system functions use connections, as well as some packages like readr.
```
Comment (jennybc) [inline content/blog/archive-1-1-0/index.Rmd:59]:
```suggestion
`archive_read()` is used to read a single file from an archive. This function returns an R connection, which can be passed to many R functions that take a connection object as input. All base R file system functions use connections, as well as some packages like readr.
```

**Comment by jimhester** [comment]
Thank you both for reviewing!

**Pull request merged**